conf/websocket.ini

```diff
@@ -38,6 +38,12 @@ handlers = console
 qualname = sentry.errors
 propagate = 0
 
+[logger_ws4py]
+level = WARNING
+qualname = ws4py
+handlers = console
+propagate = 0
```

Comment:
I suggest adding a brief explanatory note here, eg. `propagate = 0 # prevent reporting to Sentry`?

Reply:
Good suggestion. One minor point: I found out after [some experimenting](https://gist.github.com/fatbusinessman/72f47a76da5307303d34ffa9d94308b6) that [Pyramid’s logging config file format](https://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/logging.html#logging-configuration) doesn’t support inline comments. I added the comment on its own line instead.